modules/installation-registry-storage-non-production.adoc

```diff
@@ -1,10 +1,11 @@
 // Module included in the following assemblies:
 //
+// * installing/installing_aws_user_infra/installing-aws-user-infra.adoc
 // * installing/installing_bare_metal/installing-bare-metal.adoc
 // * installing/installing_vsphere/installing-vsphere.adoc
 
 [id="installation-registry-storage-non-production_{context}"]
-= Provide storage to the image registry
+= Providing non-production storage to the image registry
```

Comment:
This title and section seem to be about configuring an image registry to emptydir for non-production clusters. As such, the wording of the title is a little confusing to me at present. Would it make sense to rename this to be something similar to:

> Configuring storage for the image registry in non-production clusters

I think the `non-production storage` bit is what's bothering me.

Reply:
Or possibly:
> Configuring an empty directory for non-production clusters

Reply:
I like the way the first one sounds. I'll update the file. Thank you!